my-code/src/components/empty/empty.story.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import Empty from './index'
+
+const stories = storiesOf('Component Empty', module)
+
+stories.add('Empty Default', () => (
+  <div style={{ background: '#0a1014', fontFamily: 'Roboto', padding: 20, width: '100%', display: 'flex' }}>
```

Comment:
You could create a component named Container or Wrapper, than it would avoid to duplicate so many code:
```jsx
const Wrapper = ({children}) => {
   return (<div style={ ...styles}>{children}</div>)
}